src/test/feature/gtest-parallel

```diff
@@ -260,6 +325,8 @@ parser.add_option('--gtest_color', type='string', default='yes',
                   help='color output')
 parser.add_option('--gtest_filter', type='string', default='',
                   help='test filter')
+parser.add_option('--gtest_schedule', type='string', default='',
+                  help='test schedule')
 parser.add_option('--gtest_also_run_disabled_tests', action='store_true',
                   default=False, help='run disabled tests too')
 parser.add_option('--format', type='string', default='filter',
```

Comment:
Well. The help info is just a copy of option string. That is not good I think. We should make users quick and easy to understand just from the help info.